english/wake-up/wake-up-again/wake-up-again.md

```diff
@@ -10,4 +10,6 @@ figured you’d be able to go back to sleep. But not back to the nightmare
 
 you just had, obviously.
 
-As your eyes droop and you started drifting back into the other world
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.